### PR TITLE
Increase test coverage for the miecd_vietnam_branch

### PR DESCRIPTION
**Pull request by FrankApiyo**
Fixes: #65

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -61,7 +61,7 @@ REACT_APP_ONADATA_ACCESS_TOKEN_URL=https://api.ona.io/o/token/
 REACT_APP_ONADATA_AUTHORIZATION_URL=https://api.ona.io/o/authorize/
 
 REACT_APP_ONADATA_USER_URL=https://api.ona.io/api/v1/user.json
-REACT_APP_SUPERSET_ANALYSIS_ENDPOINT=https://discover.ona.io/superset/dashboard/25/?standalone=true
+# REACT_APP_SUPERSET_ANALYSIS_ENDPOINT=https://discover.ona.io/superset/dashboard/25/?standalone=true
```
Comment (moshthepitt) [inline .env.sample]: Why are you commenting this out?

Comment (FrankApiyo) [reply .env.sample]: this url has been moved to the constants file

Comment (FrankApiyo) [reply .env.sample]: So this line is commented out because we don't get this url from the `.env` file anymore, it's gotten from the constants file

Comment (moshthepitt) [reply .env.sample]: why dont you delete it then? @FrankApiyo

Comment (FrankApiyo) [reply .env.sample]: @moshthepitt. done removed it.

**Comment by moshthepitt** [comment]
We need to fix the warnings logged to the console.

**Comment by FrankApiyo** [comment]
There's a separate issue for this here: https://github.com/OpenSRP/opensrp-web/issues/64. I thought. I'd do that as part of a separate task

**Comment by moshthepitt** [comment]
ok, approved

**Pull request merged**